### ISSUE TITLE
Hilldwarf favoritism

### ISSUE THREAD

**Issue by darrensapalo** [addicteduser/dnd-hp-calc-discordbot#4]
May I ask [why there is favoritism for hill dwarves](https://github.com/addicteduser/dnd-hp-calc-discordbot/blob/4c2fed32558359631fffbadb6c5f2b259c9e352e/bot.py#L200)?

Nice bot!

**Comment by addicteduser** [comment]
> May I ask [why there is favoritism for hill dwarves](https://github.com/addicteduser/dnd-hp-calc-discordbot/blob/4c2fed32558359631fffbadb6c5f2b259c9e352e/bot.py#L200)?

Oh wow. I wasn't notified in my email about this issue XD.

[Hill Dwarves](https://www.dndbeyond.com/races/dwarf#HillDwarf) have this racial feature called **Dwarven Toughness**, wherein their hit point maximum increases by 1, and it increases by 1 every time they gain a level.

Hope this answers your question!

> Nice bot!

Thanks! I pushed some updates just this afternoon, that's why I was able to see that there was an open issue pala XD
